source/plugins/ruby/KubernetesApiClient.rb

```diff
@@ -456,6 +461,33 @@ def getContainerResourceRequestsAndLimits(pod, metricCategory, metricNameToColle
               metricProps["Collections"].push(metricCollections)
               metricItem["DataItems"].push(metricProps)
               metricItems.push(metricItem)
+              #Telemetry about omsagent requests and limits
```

Comment:
Thanks for adding this. This works and also you are not adding additional calls. For future reference, you ca easily pass this to us thru downward API. Please see here for reference - https://github.com/microsoft/Docker-Provider/blob/08a6c6ae1c0bf31b3bb10b07b9b2b0042a9978a5/otelcollector/deploy/prometheus-collector.yaml#L101